services/harmony/app/models/label.ts

```diff
@@ -246,7 +246,8 @@ export async function getLabelsForUser(
 ): Promise<string[]> {
   const query = trx(`${USERS_LABELS_TABLE}`)
     .select('value')
-    .orderBy('updatedAt', 'desc')
+    .groupBy('value')
```

Comment:
We'll probably want to run another performance test at some point with a large number of labels though not needed prior to merging this in.

Reply:
I can use the code I wrote to do the last test for that